TestTools/StreamChatTestTools/Extensions/Unique/URL+Unique.swift

```diff
@@ -26,7 +26,13 @@ public extension URL {
 
 extension URL {
     static var temporaryDirectoryRoot: URL {
-        URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
```

Comment:
Issue is resolved with this line. Seems like iOS is eager to clean up files when we create a huge file there.

Created DB files are deleted:
1. When `DatabaseContainer_Spy` `deinit` is called
2. Special clean up in `setUpWithError` in ChannelListPayloadTests (just in case measure)